Say that flit is lifted and by whom (#1322)
pypa/packaging.python.org: source/key_projects.rst

to PyPI. Flit requires Python 3, but you can use it to distribute modules for
Python 2, so long as they can be imported on Python 3.

> The flit packaged is lifted by `Matthias Bussonnier

Reviewer: ```suggestion
The flit package is lifted by `Matthias Bussonnier
```

> <https://github.com/Carreau>`__ since October 2023 on the `tidelift paltform

Reviewer: ```suggestion
<https://github.com/Carreau>`__ since October 2023 on the `tidelift platform
```

> <https://github.com/Carreau>`__ since October 2023 on the `tidelift platform
> `<https://tidelift.com/lifter/search/pypi/flit`__, and funds sent to the PSF and

Reviewer 2: ```suggestion
<https://github.com/Carreau>`__ since October 2023 on the `tidelift platform
<https://tidelift.com/lifter/search/pypi/flit>`__, and funds sent to the PSF and
```

> `<https://tidelift.com/lifter/search/pypi/flit`__, and funds sent to the PSF and
> earmarked for PyPA usage.

Reviewer 2: ```suggestion
`<https://tidelift.com/lifter/search/pypi/flit`__, and funds sent to the PSF are
allocated for PyPA usage.
```

Reviewer 3: @pfmoore @Carreau @pradyunsg could you check out this wording suggestion?

Reviewer: Works for me, but I’m not a lifter for flit so it’s not really my call beyond being ok with the grammar.
